js/cedar.js

```diff
@@ -69,8 +65,8 @@ Cedar.Application = function(options) {
   Cedar.config.wait = this.options.wait;
   Cedar.config.fetch = this.options.fetch;
 
-  if (typeof Cedar.$ === "undefined") {
-    Cedar.$ = jQuery({});
+  if (Cedar.events === undefined) {
```

Comment:
If you are checking a variable that doesn't exist and has never been defined, you can't do the above because you will get an error.

For example this will produce an error:
```
if (totallyNewVariable === undefined)
```

But this will work:
```
var totallyNewVariable;
if (totallyNewVariable === undefined)
```